README.rst

```diff
@@ -90,3 +90,76 @@ Architecture
      |
      |
      v
+
+
+FAQ
+------------
+
+Q: If the worker and backend complain about ::
+
+    self.db[key] = value
+    TypeError: String or Integer object expected for key, unicode found
+
+A: Modify the config for `fedmsg.d/fmn.py` to use redis
+
+| diff --git
+| config = {
+|      "fmn.rules.utils.use_pkgdb2": False,
+|      "fmn.rules.utils.pkgdb2_api_url": "http://209.132.184.188/api/",
+|      "fmn.rules.cache": {
+| -        "backend": "dogpile.cache.dbm",
+| -        "expiration_time": 300,
+| -        "arguments": {
+| -            "filename": "/var/tmp/fmn-cache.dbm",
+| +        'backend': 'dogpile.cache.redis',
+| +        'arguments': {
+| +            'host': 'localhost',
+| +            'port': 6379,
+| +            'db': 0,
+| +            'redis_expiration_time': 60*60*2,   # 2 hours
+| +            'distributed_lock': True
+|          },
+| +        # "backend": "dogpile.cache.dbm",
+| +        # "expiration_time": 300,
+| +        # "arguments": {
+| +        #     "filename": "/var/tmp/fmn-cache.dbm",
+| +        # },
+|      },
+
+Q: When I run the worker and backend it constantly quits right away
+
+A: Do you have `rabbitmq-server` and `redis` running? ::
+
+    systemctl start rabbitmq-server
```

Comment:
You can do both in one line: ``systemctl start foo bar`` so you could do status first, then start